mailchimp.rb

```diff
@@ -0,0 +1,100 @@
+require "rest-client"
+require "json"
+
+class MailChimp
+
+    # def initialize(data_center, api_version, language)
+    #     DATA_CENTER = data_center
+    #     API_VERSION = api_version
+    #     LANGUAGE = language
+    # end
+        
+    class Api
+        APP = RestClient
+        DATA_CENTER = "us19"
+        API_VERSION = "3.0"
+        AUTH = {
+            Authorization: "Bearer " + ENV["MC_AUTH_TOKEN"],
+            Content_Type: "application/json"
+        }
+        def get(path)
+            APP.get "https://#{DATA_CENTER}.api.mailchimp.com/#{API_VERSION}/#{path}", AUTH    
+        end
+        def post(path, payload)
+            APP.post "https://#{DATA_CENTER}.api.mailchimp.com/#{API_VERSION}/#{path}", payload, AUTH
+        end
+    end
+
+    class User
+
+        def get_contact_info()
+            api = Api.new
```

Comment:
i like to lean on class methods to  prevent having to do `new` and then an instance method.

So you can change 
`
api = Api.new
api.get
`

`Api.get()`

The only time i don't recommend this is when i have something i 'm going to be using multiple times. In this instance we only seem to be using once